stable/gangway/templates/key.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if not .Values.env.GANGWAY_SESSION_SECURITY_KEY -}}
```

Comment:
This breaks the chart with error: `Error: Failed to render chart: exit status 1: Error: template: gangway/templates/key.yaml:1:18: executing "gangway/templates/key.yaml" at <.Values.env.GANGWAY_SESSION_SECURITY_KEY>: can't evaluate field GANGWAY_SESSION_SECURITY_KEY in type interface {}`

Reply:
Sorry, I have passed `env: []` by mistake instead of the default `env: {}`. This works fine.